tests/dialects/test_oracle.py

```diff
@@ -57,8 +58,17 @@ def test_oracle(self):
 
     def test_join_marker(self):
         self.validate_identity("SELECT e1.x, e2.x FROM e e1, e e2 WHERE e1.y (+) = e2.y")
-        self.validate_identity("SELECT e1.x, e2.x FROM e e1, e e2 WHERE e1.y = e2.y (+)")
-        self.validate_identity("SELECT e1.x, e2.x FROM e e1, e e2 WHERE e1.y (+) = e2.y (+)")
```

Comment:
This test was incorrect, using `(+)` in both operands isn't a valid form in Oracle SQL AFAICT.